### PR TITLE
Add suse-build-key test for JeOS

### PR DESCRIPTION
**Pull request by ccret**
Add test for suse-build-key, checking that the package is installed and the key is not empty

- Related ticket: https://progress.opensuse.org/issues/45005
- Verification run: http://ccret.suse.cz/tests/2898#

### CODE REVIEW

File: tests/jeos/build_key.pm [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+sub run {
+    # check that suse-build-key is installed
+    my $zypper_output = script_output('zypper search suse-build-key');
+    die 'suse-build-key is not installed' unless $zypper_output =~ /i\+(.*)suse-build-key/;
+
+    assert_script_run('ls /usr/lib/rpm/gnupg/keys | grep suse_ptf_key.asc');
+
+    # check that the key is not empty or has invalid content
+    my $key_content = script_output("cat /usr/lib/rpm/gnupg/keys/suse_ptf_key.asc");
```
Comment (asmorodskyi) [inline tests/jeos/build_key.pm]: sure thing your version will work too but still you can simplify it with 
```suggestion
validate_script_output("cat /usr/lib/rpm/gnupg/keys/suse_ptf_key.asc", sub {
       /(----BEGIN PGP PUBLIC KEY BLOCK-----)\s*|(.*)(-----END PGP PUBLIC KEY BLOCK-----)/
});
```

Comment (ccret) [reply tests/jeos/build_key.pm]: Updated.
New validation run: http://ccret.suse.cz/tests/2899#step/build_key/7
